Userland/Libraries/LibGUI/Variant.cpp

```diff
@@ -43,6 +43,11 @@ Variant& Variant::operator=(JsonValue const& value)
         return *this;
     }
 
+    if (value.is_double()) {
+        set(static_cast<float>(value.as_double()));
```

Comment:
Why cast a `double` to `float` instead of extending `GUI::Variant` with `double`?

Reply:
Good point, that results in much nicer code